AMW_angular/io/src/app/deployments/deployments.component.ts

```diff
@@ -289,22 +296,22 @@ export class DeploymentsComponent implements OnInit {
     );
   }
 
-  copyURL() {
+  showToast: boolean = false;
```

Comment:
Like in java classes, property are usually declared at the top and not in between methods. Just move it up to the top of the class